src/TcpConnector.php

```diff
@@ -11,10 +11,11 @@
     private $loop;
     private $context;
 
-    public function __construct(LoopInterface $loop, array $context = array())
+    public function __construct(LoopInterface $loop, array $context = array(), array $options = array())
```

Comment:
Not a big fan of these "options" tbh, how about transporting this as part of "context"? Note that this would also need to be covered in our README for `Connector`, `TcpConnector`, `Server` and `TcpServer` at least.